templates/jvm.options.dockstore.es.template

```diff
@@ -33,9 +33,9 @@
 ################################################################
 
 ## GC configuration
--XX:+UseConcMarkSweepGC
--XX:CMSInitiatingOccupancyFraction=75
--XX:+UseCMSInitiatingOccupancyOnly
+8-13:-XX:+UseConcMarkSweepGC
```

Comment:
I don't think you want to change this file -- these are options for the Dockstore VM, not the ES VM. At least based on the name.

Reply:
wasnt sure either but `docker-compose.yml` points towards this file for the ES container

Reply:
> wasnt sure either but `docker-compose.yml` points towards this file for the ES container

Ah, sorry, you're right! I saw "dockstore" in the filename and overlooked "es". Looking at it more closely, it seems you shouldn't change the other file then -- that is used in docker-compose.dev.yml, which is  currently not running, but if it is, uses ES 6.6